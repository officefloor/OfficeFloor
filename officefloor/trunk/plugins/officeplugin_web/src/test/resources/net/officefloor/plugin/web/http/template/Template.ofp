<html>
<body>
Template ${TemplateName}
<br />
<!-- {BeanTree} -->
Provide ${nullBean not to be displayed $} ${bean BEAN ${value} $}
<br />
<!-- {NullBean} -->
This content not rendered as null bean provided.
<br />
<!-- {NoBean} -->
with a listing of <a href="#{beans}">beans</a>:
<br />
<table>
    <tr>
        <td>Name</td>
        <td>Description</td>
        <td>Property toString</td>
        <td>Property Value</td>
    </tr>
    <!-- {List} -->
    <tr>
        <td>${Name}</td>
        <td>${description}</td>
        <td>${Property}</td>
        <td>${property.value}</td>
    </tr>
    <!-- {Tail} -->
</table>
<form action="#{submit}">
    <input type="submit" />
</form>
</body>
</html>