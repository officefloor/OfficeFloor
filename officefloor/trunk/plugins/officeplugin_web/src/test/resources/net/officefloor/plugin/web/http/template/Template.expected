<html>
<body>
Template Test
<br />

Provide  BEAN value=tree
<br />

with a listing of <a href="/uri-beans${URI_SUFFIX}">beans</a>:
<br />
<table>
    <tr>
        <td>Name</td>
        <td>Description</td>
        <td>Property toString</td>
        <td>Property Value</td>
    </tr>
    
    <tr>
        <td>one</td>
        <td>Same</td>
        <td>toString=A</td>
        <td>value=A</td>
    </tr>
    
    <tr>
        <td>two</td>
        <td>Child</td>
        <td></td>
        <td></td>
    </tr>
    
</table>
<form action="/uri-submit${URI_SUFFIX}">
    <input type="submit" />
</form>
</body>
</html>